INSTALL.md

```diff
@@ -26,7 +26,9 @@ The following libraries are required:
 * libmicrohttpd
 * libxi, libxtst, qtx11extras (optional for auto-type on X11)
 * libsodium (>= 1.0.12, optional for KeePassXC-Browser support)
-* libargon2
+* argon2
+* qrencode
+* yubikey ykpers
```

Comment:
yubikey is not required, it is optional

Reply:
thank you.  note that https://github.com/keepassxreboot/keepassxc/wiki/Set-up-Build-Environment-on-OS-X shows it is required.